lib/src/commands/OptionsProcessor.ts

```diff
@@ -115,7 +116,20 @@ export class OptionsProcessor {
 
   private processColor(key: string, value: any, options: Record<string, any>) {
     if (isEqual(key, 'color') || endsWith(key, 'Color')) {
-      options[key] = value === null ? 'NoColor' : this.colorService.toNativeColor(value);
+      const newColorObj: Record<string, any> = { dark: 'NoColor', light: 'NoColor' };
+      if (!value) {
```

Comment:
```suggestion
      if (value === null) {
```

I think we should check here if `value === null` because if it's undefined it shouldn't have any effect in native. passing `NoColor` will reset the current color.